include/cipher/aes_gcm.h

```diff
@@ -52,7 +52,7 @@ int aes_gcm_encrypt(unsigned char *key,
                     size_t key_len,
                     unsigned char *inData,
                     size_t inData_len, unsigned char **outData,
-                    size_t * outData_len);
```

Comment:
There are a lot of changes that are only white space changes. These should not be part of the PR.